libcontainerd/client_windows.go

```diff
@@ -157,6 +158,13 @@ func (clnt *client) Create(containerID string, spec Spec, options ...CreateOptio
 		}
 	}
 
+	servicingContainerType := "Servicing" // Corresponds to enum names.
+	for _, option := range options {
+		if s, ok := option.(*ServicingOption); ok && s.IsServicing {
+			cu.ContainerType = servicingContainerType
+		}
+	}
```

Comment:
Do you also need to single out the servicing option in the other options/Apply loop?